libsplinter/src/admin/store/diesel/operations/update_circuit.rs

```diff
@@ -61,6 +61,7 @@ impl<'a> AdminServiceStoreUpdateCircuitOperation
                     circuit::durability.eq(circuit_model.durability),
                     circuit::routes.eq(circuit_model.routes),
                     circuit::circuit_management_type.eq(circuit_model.circuit_management_type),
+                    circuit::circuit_status.eq(circuit_model.circuit_status),
```

Comment:
Does this need to be added below  to the sqlite implementation as well?

Reply:
Yes it does! thanks for noticing

Reply:
Updated